### ISSUE TITLE
Firebase intergration

### ISSUE THREAD

**Issue by kanego12** [andreciornavei/flutter-getx-concept#1]
Hello sir,

Can you please include firebase integration to the project? its really the one to help build many dreams.

**Comment by DebjyotiKumarGupta** [comment]
Yes I can integrate Firebase , just create an account and provide me the access , Email - karanguptadev123@gmail.com ,
